ui/admin/app/components/form/onboarding/create-resources/index.hbs

```diff
@@ -0,0 +1,52 @@
+<Rose::Form
+  class='full-width'
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @onSubmit={{fn @submit this.hostAddress this.targetPort}}
+  as |form|
+>
+
+  <form.checkbox
+    @label={{t 'onboarding.quick-start.titles.toggle-resource-info'}}
+    @onChange={{this.toggleTargetSetup}}
```

Comment:
There's an asymmetry in the naming between `toggleTargetSetup` and `showHostInfo`.